src/js/rx/reducers/prescriptions.js

```diff
@@ -1,11 +1,37 @@
 import set from 'lodash/fp/set';
+import _ from 'lodash';
 
 const initialState = { items: [] };
 
+function sortByName(obj) {
+ /*
+  Making all values the same case, to prevent
+  alphabetization from getting wonky.
+  */
+  return obj.attributes['prescription-name'].toLowerCase();
+}
+
+function sortByFacilityName(obj) {
+  return obj.attributes['facility-name'];
+}
+
+function sortByLastRequested(obj) {
+  return new Date(obj.attributes['refill-date']).getTime();
+}
+
 export default function prescriptions(state = initialState, action) {
   switch (action.type) {
     case 'LOAD_PRESCRIPTIONS_SUCCESS':
       return set('items', action.data.data, state);
+    // After the data is loaded, we can just use `state`.
+    // Also breaking convention and using lower case because the query parameters
+    // are lower case.
```

Comment:
I _don't actually care_ about breaking the convention if the code works. I documented the choice to break the convention because someone will ask why.